openid-sharedsignals-framework-1_0.md

```diff
@@ -1732,11 +1793,9 @@ state
 > OPTIONAL An opaque value provided by the Event Receiver when the event is
   triggered. This is a nested attribute in the event payload.
 
-subject
+sub_id
```

Comment:
Same comment as above

Reply:
This is a field in the verification event. On the call on 7/18, we agreed that the events should contain "sub_id" claims, but the requests may, where appropriate, use the term "subject"

Reply:
Oh, you are right! We should move this field out of the Verification event schema, since it is supposed to be in the top level of the SET. We just need a note somewhere indicating how to specify the `sub_id` in the SET. This makes it look like the SET should be

```
{
  "jti": "123456",
  "iss": "https://transmitter.example.com",
  "aud": "receiver.example.com",
  "iat": 1493856000,
  "sub_id": {
    "format": "opaque",
    "id": "f67e39a0a4d34d56b3aa1bc4cff0069f"
  },
  "events": {
    "https://schemas.openid.net/secevent/ssf/event-type/verification":{
      "state": "VGhpcyBpcyBhbiBleGFtcGxlIHN0YXRlIHZhbHVlLgo=",
      "sub_id": {
          "format": "opaque",
          "id": "f67e39a0a4d34d56b3aa1bc4cff0069f"
       }
    }
  }
}
```
when it should really just be
```
{
  "jti": "123456",
  "iss": "https://transmitter.example.com",
  "aud": "receiver.example.com",
  "iat": 1493856000,
  "sub_id": {
    "format": "opaque",
    "id": "f67e39a0a4d34d56b3aa1bc4cff0069f"
  },
  "events": {
    "https://schemas.openid.net/secevent/ssf/event-type/verification":{
      "state": "VGhpcyBpcyBhbiBleGFtcGxlIHN0YXRlIHZhbHVlLgo="
    }
  }
}
```
like we show in the normalized examples below.

Reply:
Updated text.